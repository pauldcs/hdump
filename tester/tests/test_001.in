rss/urandom_2546955 --size=10