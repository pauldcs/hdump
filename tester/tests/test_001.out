0000000000:  a3 4e 6a 31 a3 98 02 5c 6b d8                    .Nj1...\k.

